### ISSUE TITLE
CAP 11 - ART 73 - PAR 2 - Caneleira

### ISSUE THREAD

**Issue by gabrieljm00** [CO-Intercomp/estatuto-intercomp#24]
Caneleira OBRIGATÓRIA, não apenas recomendada

**Comment by LucasEPO** [comment]
Discordo, acho que legal ter mais uma ferramenta de segurança mas isso é forçar ou quem jogar ter dinheiro para comprar, ou a atletica ter esse dinheiro, e lembrando q as atleticas vem de muito tempo parada pela pandemia.
Outro ponto é que teria que ser feito a verificação pelo representante, o que pode gerar atrasos e conflitos, se visto apenas por cima nada impede de ser apenas papelao.
Terceiro ponto é que muitos dos jogadores estão acostumados a jogar, quem joga sabe se é capaz de jogar com ou sem caneleira, então acredito que deixar recomendado legal, porque quem quer usa, quem sabe q não precisa, ou nem tem foco de jogar o futsal ou não tem como investir pq so vai jogar no inter pra completar nao precisa investir uma grana nisso
